src/test/app/claims/converters/claimantResponseConverter.ts

```diff
@@ -78,25 +83,35 @@ const courtPaymentIntentionBySetDate = {
   }
 }
 
+const courtDecisionBySetDate = {
+  'courtDecision': {
+    'paymentOption': 'BY_SPECIFIED_DATE',
+    'paymentDate': MomentFactory.currentDate().add(5, 'days'),
```

Comment:
same as above. Also this may need to be changed in the other tests. This is my first time looking at these.